addons/website_slides/views/website_slides_templates_utils.xml

```diff
@@ -38,17 +48,19 @@
 <template id="slide_share_modal_header">
     <div class="modal-header">
         <h5 class="modal-title" id="slideChannelShareModalLabel">
-            Share
+            <t t-if="share_slide_modal_title" t-out="share_slide_modal_title"/>
+            <t t-else="">Share This Course</t>
```

Comment:
Side-note, looks like it says "Share This Course" when sharing a slide, not sure if intended or not:

![image](https://user-images.githubusercontent.com/41282536/181469598-92e4ccbd-5ed2-498f-b3d6-cc14590b9a9e.png)